browser/resources/settings/brave_overrides/about_page.ts

```diff
@@ -30,6 +30,7 @@ RegisterPolymerTemplateModifications({
       }
       const parent = version.parentNode
       const wrapper = document.createElement('a')
+      version.setAttribute('style', 'color: var(--cr-link-color)')
```

Comment:
Hey @XavierJDN, thanks for working on this. I think it would be better if instead of modifying the style attribute we add some CSS for overriding it (you should be able to add the CSS in the `<style>` tag on line 15 above).